src/helpers/openFileDialog.ts

```diff
@@ -12,9 +15,18 @@ export function openFileDialog(accept: string, multiple: boolean, callback: (arg
   // Accept multiple files
   inputElement.multiple = multiple;
   // set onchange event to call callback when user has selected file
-  inputElement.addEventListener('change', callback);
+  //inputElement.addEventListener('change', callback);
+  inputElement.addEventListener('change', arg => {
+    callback(arg);
+    // remove element
+    document.body.removeChild(inputElement);
+  });
   // set onblur event to call callback when user has selected file on Safari
-  inputElement.addEventListener('blur', callback);
+  inputElement.addEventListener('blur', arg => {
+    callback(arg);
+    // remove element
+    document.body.removeChild(inputElement);
+  });
```

Comment:
Can we remove it since we have another fix from now on?

Reply:
@alessandro-bottamedi If everything works fine then please remove this blur event listener, and we're going to merge this PR ;)

Reply:
Lines removed ;)